### DISCUSSION TITLE
For people who have Sys_Crash problem

### DISCUSSION THREAD

**Discussion by magpie-o** [sultim-t/xash-rt#145] [General]
Hello, everyone. I cannot open this mod because I receive Sys_Crash problem. I have RTX 3050, it is supposed to run, but it doesn't.

I have an idea - let's share our thoughts about this problem here and try to deal with it together. As I can see, a lot of people have this problem.

**Comment by Toughbook-CFC2** [comment]
![image](https://user-images.githubusercontent.com/74775077/225154153-5d3b31b3-d38b-4746-aca3-8c4c0e03eaf3.png)

I get this sys crash error message when trying to run this under Arch on an AMD Vega M GH with lutris using the settings shown here to work 
at least for steam deck owners.
